source/install/common-prod-deploy-tar.rst

```diff
@@ -19,27 +19,32 @@ These instructions outline how to install Mattermost Server on a 64-bit Linux ho
 
 **Deploy Generic Linux**
 
-1. Log in to the server that will host Mattermost Server and open a terminal window.
+1. Install and configure a MySQL or PostgreSQL database. Refer to one of the guides below to deploy the database based on your operating system:
 
-2. Download `the latest version of the Mattermost Server <https://mattermost.com/deploy/>`__. In the following command, replace ``X.X.X`` with the version that you want to download:
+   - `Ubuntu </install/installing-ubuntu-2004-LTS.html#install-a-database>`__
+   - `Debian </install/install-debian.html#install-a-database>`__
```

Comment:
Can we add RHEL / CentOS here too? 

```
- `RHEL / CentOS </install/install-rhel-8.html#install-a-database>`__
```

Reply:
I certainly can! And have in a new commit.